dev/Detours/disasm.cpp

```diff
@@ -294,21 +294,21 @@ class CDetourDis
     static BOOL             s_fLimitReferencesToModule;
 
   protected:
-    BOOL                m_bOperandOverride;
-    BOOL                m_bAddressOverride;
-    BOOL                m_bRaxOverride; // AMD64 only
-    BOOL                m_bVex;
-    BOOL                m_bEvex;
-    BOOL                m_bF2;
-    BOOL                m_bF3; // x86 only
-    BYTE                m_nSegmentOverride;
-
-    PBYTE *             m_ppbTarget;
-    LONG *              m_plExtra;
-
-    LONG                m_lScratchExtra;
-    PBYTE               m_pbScratchTarget;
-    BYTE                m_rbScratchDst[64]; // matches or exceeds rbCode
+    BOOL                m_bOperandOverride{ false };
```

Comment:
I see at least some of these get initialized in the constructor.  I don't know this code, but in general I don't think we want to initialize it both places.

Reply:
Can you find the owners of detours to see how they'd want to handle this?

Reply:
https://github.com/microsoft/Detours/blob/main/src/image.cpp
They actually haven't handle this at all

Reply:
I think I will let this slide and set it as the baseline for prefast